tests/unit/test_zos_console.py

```diff
@@ -19,4 +20,12 @@ def setUp(self):
     def test_object_should_be_instance_of_class(self):
         """Created object should be instance of Console class."""
         console = Console(self.session_details)
-        self.assertIsInstance(console, Console)
\ No newline at end of file
+        self.assertIsInstance(console, Console)
+    
+    @mock.patch('requests.Session.send')
+    def test_get_response_should_return_messages(self, mock_send_request):
+       """Getting z/OS Console response messages on sending a response key"""
+       mock_send_request.return_value = mock.Mock(headers={"Content-type": "application/json"}, status_codes=200)
+
+       Console(self.session_details).get_response(self)
```

Comment:
The response key should be a string like `"console-key"` instead of the class instance `self`.

Reply:
okay thanks

Reply:
thank you now the tests are passing

Reply:
also how can I amend the last commit?

Reply:
You should be able to amend the commit using your preferred Git tool, and then force push the changes to this branch.

I believe the Git CLI commands would be:
```
git commit --amend -m "changes" -s
git push -u origin get_command1 --force-with-lease
```